providers/class-two-factor-totp.php

```diff
@@ -274,7 +280,7 @@ public static function is_valid_authcode( $key, $authcode ) {
 		 *
 		 * @param int $max_ticks Max ticks of time correction to allow. Default 4.
 		 */
-		$max_ticks = apply_filters( 'two-factor-totp-time-step-allowance', self::DEFAULT_TIME_STEP_ALLOWANCE );
+		$max_ticks = apply_filters( 'two_factor_totp_time_step_allowance', self::DEFAULT_TIME_STEP_ALLOWANCE );
```

Comment:
This should also remain unchanged if we want to avoid breaking existing setups.

Reply:
Given we're still at 0.x.x, let's fix this now (while people expect it to break) and add a note to the readme.

Reply:
This plugin is used on WP VIP and 10k+ other instances as reported by WP.org. I would strongly suggest that we don't break existing installations because of a naming preference for core hooks.

Changing this for a merge into core would be totally OK, though.

Another alternative is to introduce a "deprecated" file which takes care of these legacy setups. I'm not sure that's worth the effort for this cleanup.

Reply:
How many of those folks are using the filter? Probably a small fraction. (We don't have telemetry.)

Reply:
Yeah, we'll never know the exact numbers. I still think we shouldn't include this breaking change in a cleanup pull request.

Reply:
Done.